ports/servoshell/tracing.rs

```diff
@@ -0,0 +1,229 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/// Log an event from winit at trace level.
+/// - To disable tracing: RUST_LOG='servoshell<winit@=off'
+/// - To enable tracing: RUST_LOG='servoshell<winit@'
+/// - Recommended filters when tracing is enabled:
+///   - servoshell<winit@DeviceEvent=off
+///   - servoshell<winit@MainEventsCleared=off
+///   - servoshell<winit@NewEvents(WaitCancelled)=off
+///   - servoshell<winit@RedrawEventsCleared=off
+///   - servoshell<winit@RedrawRequested=off
+///   - servoshell<winit@UserEvent(WakerEvent)=off
+///   - servoshell<winit@WindowEvent(AxisMotion)=off
+///   - servoshell<winit@WindowEvent(CursorMoved)=off
+macro_rules! trace_from_winit {
```

Comment:
Maybe "trace_winit_event" would be a little clearer?

Reply:
done in 198decce710d39b80ab8e02b35396515145186b5